tests/test_eratos_sdk.py

```diff
@@ -56,3 +56,22 @@ def test_silo_time_drill(self):
         self.assertEqual(bars_silo['time'].data[-1], np.datetime64('2023-06-10T00:00:00.000000000'))
         self.assertEqual(xr.core.formatting.first_n_items(bars_silo.max_temp, 1), 18.4)
         self.assertEqual(bars_silo['max_temp'].shape, (9,))
+
+    def test_silo_time_drill_no_import(self):
+        """
+        Open remote SILO dataset, slice to known spatial area, do not load eratos_xarray directly
+        :return:
+        """
+        
```

Comment:
Bit of a nit but it would be good to reduce the amount of whitespace now that the import statement is removed.
```suggestion
```